### ISSUE TITLE
Revert back to original Icon

### ISSUE THREAD

**Issue by wwwdata** [tastelessjolt/flutter_dynamic_icon#2]
**Is your feature request related to a problem? Please describe.**
I would like to be able to revert back to the original Icon

**Describe the solution you'd like**
We need a method that reverts the custom Icon

Afaik this should be possible by calling
```
UIApplication.sharedApplication setAlternateIconName:iconName
```
and setting the iconName to `nil`

I would suggest to add another flutter method that does this call with nil

**Comment by tastelessjolt** [comment]
Well, what happens if you just pass `null` from the Flutter since it'll be converted into `nil` on the iOS side?

![image](https://user-images.githubusercontent.com/16670547/61167965-f3138980-a563-11e9-9f12-1dbeef2e0654.png)

**Comment by wwwdata** [comment]
Passing null from the flutter side did nothing. Maybe there is something that prevents the iOS implementation from getting `nil` and passing that to the native API?

**Comment by tastelessjolt** [comment]
Then, I'll try to fix that (passing `null` for reverting) and then add it in the docs.

**Comment by iballan** [comment]
Hello @tastelessjolt  
Is there any news about this issue ?

**Comment by tastelessjolt** [comment]
I'm sorry but I'll try to fix up this weekend if I have time. 
But if anyone wants to take up fixing this, please request a PR. 
It's a quite simple change I think.

**Comment by tastelessjolt** [comment]
Fixed in `1.1.0`
